browser/ui/webui/playlist_ui.cc

```diff
@@ -50,6 +50,12 @@ class UntrustedPlayerUI : public ui::UntrustedWebUIController {
         network::mojom::CSPDirectiveName::StyleSrc,
         std::string("style-src chrome-untrusted://resources "
                     "chrome-untrusted://brave-resources 'unsafe-inline';"));
+    source->OverrideContentSecurityPolicy(
+        network::mojom::CSPDirectiveName::ImgSrc,
+        std::string("img-src 'self' chrome-untrusted://resources;"));
```

Comment:
Can you explain a bit further about the reasoning for utilizing `resources` here rather than the `playlist` frame? I'd be a bit concerned that storing these external resources even if they're sanitized under this origin because it's [used](https://github.com/search?q=repo%3Abrave%2Fbrave-core%20chrome-untrusted%3A%2F%2Fresources&type=code) by other parts of the browser in a way that likely doesn't assume that these are resources that weren't bundled with the browser. 

It would be ideal if we could isolate bundled resources and sanitized resources from each other under separate origins such as `chrome-untrusted://sanitized-resources` and keep `chrome-untrusted://resources` for only bundled resources.

Reply:
Here, UntrustedPlayerUI doesn't load any external `images` as it only need some button icons. 

---

> I'd be a bit concerned that storing these external resources even if they're sanitized under this origin

The sanitized images are not served via `chrome-untrusted://resources`. They are served via `chrome-untrusted://playlist-data//playlist-data/{thumbnail/media}` and playlist data source will serve matching files from user profile directory. They are not bundled with our browser but stored in the directory on demand.

Reply:
Awesome, thanks for clarifying. In that case, these all appear to be bundled resources so no concerns with this change to the CSP.